docs/Limitations.md

```diff
@@ -0,0 +1,19 @@
+# Limitations of Azure DevOps Demo Generator
+
+Azure DevOps Demo Generator is to create a demo set up with predefined templates, it has some limitations, which are listed below. This can't be treated as migration or copying project tool from one Organization to another.
+
+## Following are the limitations of Azure DevOps Demo Generator and Template Builder
+
+|||
+|------|-----|
+|Custom process template|This is supported to an extent, but not all the options available as compared to standard process template. Organization must contains same process template Id|
+| Iterations| Not all the iterations can be exported by the Template Builder with hierarchy|
```

Comment:
What does "not all" mean?  Are we saying that only a specific subset are exported?  Perhaps a hierarchy depth?  Or only the built-in iterations?